components/raftstore/src/coprocessor/config.rs

```diff
@@ -100,39 +102,52 @@ impl Default for Config {
 }
 
 impl Config {
+    pub fn region_split_size(&self) -> ReadableSize {
+        self.region_split_size
+            .unwrap_or(if self.enable_region_bucket {
```

Comment:
The preference over `LARGE_REGION_SPLIT_SIZE_MB` or `MULTI_ROCKS_SPLIT_SIZE_MB` is confusing. Needs some comment to say both of these values are used for v1 architecture only.